frontend/src/metabase/querying/filters/components/FilterPicker/FilterPicker.tsx

```diff
@@ -17,6 +20,11 @@ export interface FilterPickerProps {
 
   onSelect: (filter: Lib.Filterable) => void;
   onClose?: () => void;
+
+  columnPickerProps?: Pick<
+    FilterColumnPickerProps,
+    "withColumnItemIcon" | "withColumnGroupIcon" | "withCustomExpression"
+  >;
```

Comment:
Allow the SDK to enable/disable column item icon, group item icon, and column expression

Reply:
i like this idea but can we inline the props instead of putting another config object? It'll make parsing this code a lot easier for newcomers

Reply:
@oisincoveney ah, I was putting it in `columnPickerProps` as I thought it'd be helpful to know that these option only applies the column picker, but now that I think about it that seems like an implementation detail. I'll inline it then, thanks!

Reply:
inlined in https://github.com/metabase/metabase/pull/49768/commits/69abc1e99c4cf4c916389861c4d3f95f0e38568c